resources/assets/coffee/_classes/forum-cover.coffee

```diff
@@ -62,24 +62,23 @@ class @ForumCover
     @uploadButton.length > 0
 
 
-  toggleModal: (e) =>
-    e.preventDefault()
+  isModalOpen: (isModalOpen) =>
+    return false if !@hasCoverEditor()
 
-    if @header[0]._open
-      @closeModal()
-    else
-      @openModal()
+    if isModalOpen?
```

Comment:
? Not sure what you're asking? If it's `isModelOpen == false` then the `data-modal-open` will be emptied (the `: void 0` added by coffeescript) and `@isModalOpen()` will return false?